src/canister/individual_user_template/src/api/cdao/send_creator_dao_stats_to_subnet_orchestrator.rs

```diff
@@ -0,0 +1,31 @@
+use std::collections::HashSet;
+
+use candid::Principal;
+use ic_cdk_macros::update;
+use shared_utils::common::utils::permissions::is_caller_controller;
+
+use crate::{util::subnet_orchestrator::SubnetOrchestrator, CANISTER_DATA};
+
+#[update(guard = "is_caller_controller")]
```

Comment:
curious : Can't a update called be called from a get call ?

Reply:
inter canister callls are update calls